lib/config.js

```diff
@@ -20,7 +20,8 @@ var fs = require("fs"),
     assign = require("object-assign"),
     debug = require("debug"),
     yaml = require("js-yaml"),
-    userHome = require("user-home");
+    userHome = require("user-home"),
+    isAbsolutePath = require("path-is-absolute");
```

Comment:
Do we need a new module for this? Isn't this just checking to see if the first character is `/`?